src/domain/fiat/index.ts

```diff
@@ -0,0 +1,25 @@
+import { toSats } from "@domain/bitcoin"
+import { InvalidFiatAmount } from "@domain/errors"
+
+export const toFiat = (amount: number): FiatAmount => {
+  return amount as FiatAmount
+}
+
+export const checkedToFiat = (amount: number): FiatAmount | ValidationError => {
+  if (!(amount && amount > 0)) return new InvalidFiatAmount()
+  return toFiat(amount)
+}
+
+export const satsToFiatOptionPricing = ({
```

Comment:
This is going to be a service call. Should be represented in the domain as an `IPricingService` or something like that.

Reply:
it should be DealerService, right?

Reply:
yes fiat and dealer are really the same "thing" in the context of the galoy backend